README.md

```diff
@@ -6,6 +6,7 @@
 This is a community-driven effort to continue the development of Cortex Command.  
 Stay up to date in our [Discord channel](https://discord.gg/TSU6StNQUG).
 
+# Playing the game on Windows without building it
```

Comment:
I think we can simplify this a little to something like `How to Play the Game`.
Also, it should follow capitalization style of the rest of the headers in the readme (i.e. non-prepositions capitalized).